com.unity.render-pipelines.core/Runtime/Lighting/ProbeVolume/ProbeVolumeAsset.cs

```diff
@@ -50,6 +75,86 @@ public string GetSerializedFullPath()
+            var hasSupportData = cellSupportData.IsCreated;
+            var positionsByteCount = totalCellCounts.probesCount * UnsafeUtility.SizeOf<Vector3>();
+            var validityByteStart = AlignUp16(positionsByteCount);
+            var validityByteCount = totalCellCounts.probesCount * UnsafeUtility.SizeOf<float>();
+            var offsetByteStart = AlignUp16(positionsByteCount) + AlignUp16(validityByteCount);
+            var offsetByteCount = totalCellCounts.offsetsCount * UnsafeUtility.SizeOf<Vector3>();
+            var positionsData = hasSupportData ? cellSupportData.GetSubArray(0, positionsByteCount).Reinterpret<Vector3>(1) : default;
+            var validityData = hasSupportData ? cellSupportData.GetSubArray(validityByteStart, validityByteCount).Reinterpret<float>(1) : default;
+            var offsetsData = hasSupportData ? cellSupportData.GetSubArray(offsetByteStart, offsetByteCount).Reinterpret<Vector3>(1) : default;
+
+            var startCounts = new CellCounts();
+            for (var i = 0; i < cells.Length; ++i)
+            {
+                var cell = cells[i];
+                var counts = cellCounts[i];
+
+                cell.bricks = bricksData.GetSubArray(startCounts.bricksCount, counts.bricksCount);
+                cell.shData = shData.GetSubArray(startCounts.probesCount * CoefficientVectorsCount, counts.probesCount * CoefficientVectorsCount);
+
+                if (hasSupportData)
+                {
+                    cell.probePositions = positionsData.GetSubArray(startCounts.probesCount, counts.probesCount);
+                    cell.validity = validityData.GetSubArray(startCounts.probesCount, counts.probesCount);
+                    cell.offsetVectors = offsetsData.GetSubArray(startCounts.offsetsCount, counts.offsetsCount);
+                }
+
+                startCounts.Add(counts);
+            }
+        }
+
+        // TEMP SECTION: Manually extract a NA from bytes until we have access to: https://github.cds.internal.unity3d.com/unity/unity/pull/6458
+        bool isBlobCreated;
+        internal Dictionary<TextAsset, NativeArray<byte>> assetToBytes = new();
+        internal void ReleaseBinaryData() => OnDisable();
+        internal void LoadBinaryData() { OnEnable(); }
+        void OnEnable()
+        {
+            if (!isBlobCreated && cellDataAsset)
+            {
+                assetToBytes[cellDataAsset] = new NativeArray<byte>(cellDataAsset.bytes, Allocator.Persistent);
+
+                if(cellSupportDataAsset)
+                    assetToBytes[cellSupportDataAsset] = new NativeArray<byte>(cellSupportDataAsset.bytes, Allocator.Persistent);
+
+                isBlobCreated = true;
+            }
+        }
+
+        void OnDisable()
+        {
+            isBlobCreated = false;
+
+            foreach (var bytes in assetToBytes.Values)
+                bytes.Dispose();
+
+            assetToBytes.Clear();
+        }
+        // END TEMP SECTION
```

Comment:
So this whole block is basically just temporary scaffolding to be able to test this logic until the mentioned native PR lands allowing direct access to TextAsset data through a non-allocating NativeArray. (TextAsset.bytes makes a copy of the entire data array, which is not great)

Reply:
Removed in latest